saleor/product/search.py

```diff
@@ -26,13 +26,23 @@
 
 
 def update_products_search_vector(products: "QuerySet"):
-    products = products.prefetch_related(*PRODUCT_FIELDS_TO_PREFETCH)
-    for product in products:
-        product.search_vector = prepare_product_search_vector_value(
-            product, already_prefetched=True
+    batch_size = 300
```

Comment:
Why?

Reply:
That results in memory usage up to ~350MB from what i've tested locally, seems ok, wdyt?

Reply:
Same as above, without comments, it's just an arbitrary magic number.